modules/apps/object/object-api/src/main/java/com/liferay/object/constants/ObjectActionKeys.java

```diff
@@ -26,6 +26,8 @@ public class ObjectActionKeys {
 	public static final String EXTEND_SYSTEM_OBJECT_DEFINITION =
 		"EXTEND_SYSTEM_OBJECT_DEFINITION";
 
+	public static final String MANAGE_MODELS = "MANAGE_MODELS";
```

Comment:
Hey, I'm a little bit apprehensive about the name of this constant. What do you think about the options below?

- MODELS_MANAGER
- MODELS_MANAGEMENT

If you have any other options, feel free to share then so we can decide together the best name for this constant

Reply:
Hey Selton, `MANAGE_MODELS` was defined in mock-up.